src/main/resources/db/migration/V201807021322837__changed_analysys_fields.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE analyses ALTER COLUMN title TYPE TEXT;
```

Comment:
Why do you need TEXT for title?

Reply:
a quotation from the postgresql documentation says: "If you desire to store long strings with no specific upper limit, use text or character varying without a length specifier, rather than making up an arbitrary length limit". We have the same case because we want to use a arbitrary-length character data in the title field. I could do checks in Java code, but I find this an inconvenient solution, because it depends on the annotation(length limit) in the domain class and on the length limitation at the base level.

Reply:
This is really suspicious, I would fix validation for title field, and leave varchar there